gradle/spec/dependabot/gradle/update_checker/version_finder_spec.rb

```diff
@@ -213,6 +213,41 @@
         is_expected.to eq("https://private.registry.org/repo")
       end
 
+      context "that is a gitlab maven repository" do
+        let(:credentials) do
+          [
+            {
+              "type" => "maven_repository",
+              "url" => "https://private.registry.org/api/v4/groups/-/packages/maven/"
```

Comment:
Once a v5 is supported, it would make sense to extend to support this, given that they do not change the authorization scheme (one could hope). I can see other parts of the code also defaults to v4, like [source.rb](https://github.com/dependabot/dependabot-core/blob/3656c3de267921c038e6d23c3055941f071e87a8/common/lib/dependabot/source.rb#L131), so this would probably be part of a larger update.

There is no guarantee that the gitlab maven repository is hosted on gitlab.com, it could be any host, which is why the path is being checked.

Reply:
It all looks good to me